coordinator/gscoordinator/utils.py

```diff
@@ -830,67 +830,141 @@ def _pre_process_for_output_graph_op(op, op_result_pool, key_to_op, **kwargs):
     )
 
 
-def _pre_process_for_project_to_simple_op(op, op_result_pool, key_to_op, **kwargs):
+def _pre_process_for_project_to_simple_op(  # noqa: C901
+    op, op_result_pool, key_to_op, **kwargs
+):
     # for nx graph
     if op.attr[types_pb2.GRAPH_TYPE].graph_type in (
         graph_def_pb2.DYNAMIC_PROJECTED,
         graph_def_pb2.ARROW_FLATTENED,
     ):
         return
-    assert len(op.parents) == 1
+
+    def __check_v_prop_exists_in_all_v_labels(schema, prop):
```

Comment:
Use `_` as prefix for private functions/members. `__` in python has different implication.

See also: https://stackoverflow.com/questions/8689964/why-do-some-functions-have-underscores-before-and-after-the-function-name